instrumentation/spring/spring-rabbit-1.0/javaagent/src/test/groovy/ContextPropagationTest.groovy

```diff
@@ -94,7 +94,7 @@ class ContextPropagationTest extends AgentInstrumentationSpecification {
             "$SemanticAttributes.MESSAGING_DESTINATION_KIND" "queue"
             "$SemanticAttributes.MESSAGING_MESSAGE_PAYLOAD_SIZE_BYTES" Long
             "$SemanticAttributes.MESSAGING_RABBITMQ_ROUTING_KEY" String
-            "messaging.payload" String
+            "messaging.payload" "test"
```

Comment:
This is just a suggestion but maybe change this to "test payload" or something we can quickly understand refers to payload

Reply:
done